feature/gribi/vrf_policy_driven_te/README.md

```diff
@@ -0,0 +1,817 @@
+  ip_in_ip {
+    dst_ip: "203.10.113.2"
+    src_ip: "ipv4_outer_src_111"
+  }
+  network_instance: "TE_VRF_111"
+}
+
+NHG#200 (Default VRF) {
+  {NH#2000, DEFAULT VRF, weight:1}
+}
+NH#2000 -> {
+    network_instance: "DEFAULT"
+}
+
+IPv4Entry {203.0.113.1/32 (TE_VRF_111)} -> NHG#1 (DEFAULT VRF) -> {
+  {NH#1, DEFAULT VRF, weight:1,ip_address=192.0.2.101},
+  {NH#2, DEFAULT VRF, weight:3,ip_address=192.0.2.102},
+  backup_next_hop_group: 100 // re-encap to 203.0.113.100
+}
+IPv4Entry {192.0.2.101/32 (DEFAULT VRF)} -> NHG#2 (DEFAULT VRF) -> {
+  {NH#10, DEFAULT VRF, weight:1,mac_address:magic_mac, interface-ref:dut-port-2-interface},
+  {NH#11, DEFAULT VRF, weight:3,mac_address:magic_mac, interface-ref:dut-port-3-interface},
+}
+IPv4Entry {192.0.2.102/32 (DEFAUlT VRF)} -> NHG#3 (DEFAULT VRF) -> {
+  {NH#100, DEFAULT VRF, weight:2,mac_address:magic_mac, interface-ref:dut-port-4-interface},
+}
+
+NHG#100 (Default VRF) {
+  {NH#1000, DEFAULT VRF}
+}
+NH#1000 -> {
+  encapsulate_header: OPENCONFIGAFTTYPESENCAPSULATIONHEADERTYPE_IPV4
+  ip_in_ip {
+    dst_ip: "203.0.113.100"
+    src_ip: "ipv4_outer_src_222"
+  }
+  network_instance: "TE_VRF_222"
+}
+
+IPv4Entry {203.0.113.100/32 (TE_VRF_222)} -> NHG#7 (DEFAULT VRF) -> {
+  {NH#3, DEFAULT VRF, weight:1,ip_address=192.0.2.103},
+  backup_next_hop_group: 201 // decap to DEFAULT VRF
+}
+IPv4Entry {192.0.2.103/32 (DEFAULT VRF)} -> NHG#8 (DEFAULT VRF) -> {
+  {NH#12, DEFAULT VRF, weight:1,mac_address:magic_mac, interface-ref:dut-port-5-interface},
+}
+NHG#201 (Default VRF) {
+  {NH#2001, DEFAULT VRF, weight:1}
+}
+NH#2001 -> {
+    OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4
+    network_instance: "DEFAULT"
+}
+
+// 203.10.113.2 is the tunnel IP address. Note that the NHG#4 is different than NHG#1.
+
+IPv4Entry {203.10.113.2/32 (TE_VRF_111)} -> NHG#4 (DEFAULT VRF) -> {
+  {NH#3, DEFAULT VRF, weight:1,ip_address=192.0.2.104},
+  backup_next_hop_group: 101 // re-encap to 203.10.113.101
+}
+IPv4Entry {192.0.2.104/32 (DEFAULT VRF)} -> NHG#5 (DEFAULT VRF) -> {
+  {NH#12, DEFAULT VRF, weight:1,mac_address:magic_mac, interface-ref:dut-port-6-interface},
+}
+NHG#101 (DEFAULT VRF) {
+  {NH#1001, DEFAULT VRF}
+}
+NH#1001 -> {
+  encapsulate_header: OPENCONFIGAFTTYPESENCAPSULATIONHEADERTYPE_IPV4
+  ip_in_ip {
+    dst_ip: "203.0.113.101"
+    src_ip: "ipv4_outer_src_222"
+  }
+  network_instance: "TE_VRF_222"
+}
+IPv4Entry {203.0.113.101/32 (TE_VRF_222)} -> NHG#9 (DEFAULT VRF) -> {
+  {NH#3, DEFAULT VRF, weight:1,ip_address=192.0.2.103},
+  backup_next_hop_group: 201 // decap to DEFAULT VRF
+}
+IPv4Entry {192.0.2.103/32 (DEFAULT VRF)} -> NHG#10 (DEFAULT VRF) -> {
+  {NH#12, DEFAULT VRF, weight:1,mac_address:magic_mac, interface-ref:dut-port-7-interface},
+}
+
+```
+
+*   Install a BGP route resolved by ISIS in default VRF to rout traffic out of DUT port-8.
+
+*   Install an 0/0 static route in ENCAP_VRF_A and ENCAP_VRF_B pointing to the DEFAULT VRF.
```

Comment:
The OC Local routing model doesn't look to be having a nexthop network-instance leaf.
So should this default static route in Encap VRF be part of the DUT startup baseconfig ?